### PR TITLE
Milestone #2 : Portfolio Mobile Projects Section 

### PR DESCRIPTION
**Pull request by erickhansx**
Implemented the second part of the portfolio mobile project: 

- Added the text content and images to the portfolio projects section. 
- Implemented a CSS grid for the project to display the different sections. 
- Combined CSS grid and Flexbox inside the sections to get a better display of the site. 
- Added background colors and background images to several parts of the project. 
- Updated the readme file to include CSS grid in the Built With section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,10 +35,215 @@ <h1 class="developers-name">Charlie Giorgania</h1>
...
+                src="/resources/images/arrow.png"
+                alt="arrow image"
+                class="arrow-image"
+              />
+            </div>
+          </div>
+        </div>
+        <div>
+          <div class="about-content">
+            <h2 class="main-topic topic-h2">About</h2>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please kindly make sure that that about section uses an enclosing `section` tag.

File: index.html [hunk 2]
```diff
@@ -35,10 +35,215 @@ <h1 class="developers-name">Charlie Giorgania</h1>
...
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>CSS</li>
+              <li>Capibara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </div>
+        <div class="skills-box lang-fw-skills-boxes bottom-boxes">
```
Comment (AnselemOdims) [inline index.html]: - [ ] For semantic purposes, please kindly make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

File: index.html [hunk 3]
```diff
@@ -35,10 +35,215 @@ <h1 class="developers-name">Charlie Giorgania</h1>
...
+              <li>JavaScript</li>
+            </ul>
+            <div class="works-bttn">
+              <a href="#" class="works-btn-styles">See this project</a>
+              <img
+                src="/resources/images/arrow.png"
+                alt="arrow image"
+                class="arrow-image"
+              />
+            </div>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please kindly ensure that the project images are full width as specified in the original design

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @erickhansx , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by JuliCarracedo — APPROVED** [review]
# Status: Approved ✔️
Hi @erickhansx , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**